tsconfig.json

```diff
@@ -26,7 +26,7 @@
     "packages/building-utils/test/custom-polyfils/**/*",
     "**/dist/**/*",
     "packages/**/test/**/snapshots",
-    "packages/es-dev-server/test/fixtures/**/*",
+    "packages/es-dev-server**/*",
```

Comment:
presumably because it uses its own?
i suppose this is easier than negating the js-only options locally...

Reply:
Indeed it has it's own config